vault/audit.go

```diff
@@ -435,7 +444,11 @@ func (c *Core) setupAudits(ctx context.Context) error {
 		}
 
 		// Mount the backend
-		broker.Register(entry.Path, backend, entry.Local)
+		err = broker.Register(entry.Path, backend, entry.Local)
```

Comment:
This function can return an error -- would we want to return an error if these fail? We could use multierror here, too, but it also makes sense to try and be best effort.

Reply:
Ahh, right! Thanks for the additional context!